resources/js/Pages/Company/Project/Tasks/Partials/ProjectTaskCheckbox.vue

```diff
@@ -274,10 +290,6 @@ export default {
       this.$refs.input.focus();
```

Comment:
This doesn't work anymore.

Reply:
It should not have worked previously: there is not ":ref"

Reply:
fixed